EXERCISES/exercise20.js

```diff
@@ -0,0 +1,31 @@
+/**Crie um programa para informar quais e quantas notas são necessárias para entregar o mínimo de cédulas
+para um determinado valor informado pelo usuário considerando notas de R$ 100, R$ 50, R$ 10 e R$ 5 e R$ 1.
+Seu programa deve mostrar apenas as notas utilizadas. Por exemplo, ao solicitar R$18, o programa deve
+informar apenas a seguinte informação (note que não foram exibidas informações sobre as demais cédulas): 1
+nota(s) de R$ 10. 1 nota(s) de R$ 5. 3 nota(s) de R$ 1. */
+
+function calculaCedulas(valor) {
+  let valorInformado = isNaN(valor) ? 0 : valor;
+  let quantidadeNotas = 0, retorno = '', cedulaAtual = 0;
+  const cedulas = [100, 50, 10, 5, 1];
+
+  resto = valorInformado;
+
+  for (cedula in cedulas){
```

Comment:
```suggestion
  for (let cedula of cedulas){
```

dessa forma cedula já terá o valor, ao invés do índice do array... outra coisa, sempre que for fazer um loop, use o `let` na declaração, para limitar o escopo da variável de controle apenas para dentro do loop... sem ele, a variável de controle pode ser acessada de fora do loop, o que não é boa prática.